configure.py

```diff
@@ -2236,6 +2236,9 @@ def test_exe_extra_ldflags():
         'makefile_path': os.path.join(build_paths.build_dir, '..', 'Makefile'),
         'ninja_build_path': os.path.join(build_paths.build_dir, '..', 'build.ninja'),
 
+        # macOS does not support response files for the ar command.
```

Comment:
Are we sure other `ar`s *do* support response files? OpenBSD, AIX, Solaris, ... ? IMO might be better to just say use it if on Linux otherwise don't.